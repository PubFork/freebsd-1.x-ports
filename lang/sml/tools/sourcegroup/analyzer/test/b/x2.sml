structure A = struct val x=4 end

and B = struct val y=5 val b = A.x end
