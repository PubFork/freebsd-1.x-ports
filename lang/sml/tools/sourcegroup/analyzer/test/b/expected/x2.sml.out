source sml "b/x2.sml"
  import structure A 
  export structure A B ;

